crafting-dead-immerse/src/main/java/sm0keysa1m0n/bliss/layout/yoga/YogaLayout.java

```diff
@@ -498,6 +503,12 @@ public void markDirty() {
     Yoga.YGNodeMarkDirty(this.node);
   }
 
+  @Override
+  public void clearStyles() {
```

Comment:
Why is this required?

Reply:
When you set a layout display to none all it's children will properly update it's size due to their layout being nil, but the parent layout still have a yoga layout, and due to styles usually assigning a fixed width and height as well other styles, the parent will still be visible, by clearing the styles for when the display is set to none will effectively make it invisible since yoga will finally update the width and height to 0.
I couldn't find a way to make yoga ignore the set width and height for when the display is none